Source/Core/VideoCommon/PostProcessing.cpp

```diff
@@ -286,8 +288,10 @@ void PostProcessingConfiguration::SaveOptionsConfiguration()
     {
       std::string value;
       for (size_t i = 0; i < it.second.m_integer_values.size(); ++i)
-        value += StringFromFormat("%d%s", it.second.m_integer_values[i],
-                                  i == (it.second.m_integer_values.size() - 1) ? "" : ", ");
+      {
+        value += fmt::format("{}{}", it.second.m_integer_values[i],
```

Comment:
Would this be a potential candidate for an algorithm? `std::accumulate` comes to mind, or whatever else comes close to the C# `string.Join` method.

Reply:
I'm not sure using an algorithm here would make the readability much nicer. If the values were already strings, then the whole function could be collapsed down into a call to `JoinStrings()` in StringUtil, but they're `s32`s.